front/app/containers/Admin/projects/project/participationContext/index.tsx

```diff
@@ -69,6 +69,7 @@ export interface IParticipationContextConfig {
   input_term?: InputTerm;
```

Comment:
The `participationContext` really needs to be refactored, but it's a pretty big undertaking. I've added the new attributes to match the current implementation, but I'm not a fan of how it works today. Maybe Luuc and I can take a look at this component together at some point and at least plan out how this might be refactored?